docs/components/properties.md

```diff
@@ -645,7 +645,7 @@ export class ToDoListItem {
 
 ## Prop Validation
 
-To do validation of a Prop, you can use the [@Watch()](reactive-data/#watch-decorator) decorator:
+To do validation of a Prop, you can use the [@Watch()](./reactive-data/#watch-decorator) decorator:
```

Comment:
Does this link exist? I see `docs/reactive-data/#the-watch-decorator-watch`, but not what's listed here